rust/lance-index/src/scalar/inverted/tokenizer.rs

```diff
@@ -141,9 +145,72 @@ fn build_base_tokenizer_builder(name: &str) -> Result<tantivy::tokenizer::TextAn
             tantivy::tokenizer::RawTokenizer::default(),
         )
         .dynamic()),
+        #[cfg(feature = "tokenizer-lindera")]
+        s if s.starts_with("lindera/") => {
```

Comment:
Above, the tokenizer names start with `lindera-` not `lindera/`

Reply:
sorry, I haven't changed my unitttest yet. My idea is use a directory structure to manage different language model.

```
$LANCE_HOME/
    tokenizers/
        lindera/
            japanese-dic1/
            japanese-dic2/
            korea/
        jieba/
```